### ISSUE TITLE
ansible-lint does not detect changes in absolute paths to templates since ansible 2.0.0.2 

### ISSUE THREAD

**Issue by christx2** [ansible/ansible-lint#156]
# Issue Type
- Bug report
  Bug
- Feature request
# Ansible and Ansible Lint details

```
ansible --version
ansible-lint 2.6.2
ansible-lint --version
nsible 2.0.2.0
```
- ansible installation method: one of source, pip, OS package
  pip install on osx
- ansible-lint installation method: one of source, pip, OS package
# Desired Behavior

ansible-lint  roles verification 

Please give some details of the feature being requested
or what should happen if providing a bug report
# Actual Behaviour (Bug report only)

ansible-lint does not pick up on the absolute path changes with templates in ansible 2.0.x.x
https://github.com/ansible/ansible/issues/14161

Please give some details of what is actually happening.
Include a [minimum complete verifiable example](http://stackoverflow.com/help/mcve)
with:
- playbook
- output of running ansible-lint
  Running ansible-lint on roles i get the following:
  ansible-lint main.yml 
  [ANSIBLE0002] Trailing whitespace
  main.yml:10
  - kvm 

[ANSIBLE0002] Trailing whitespace
main.yml:11
- libvirt 

However if i alter the role and it has incorrectly set absolute paths, the ansible-lint does not give any warnings, as this has changed in the latest ansible version to adhere to jinja2 template rules.
# tasks file for vac
# 
- name: Resize filesystem
  shell: "resize2fs `df -T | awk '{ if ($NF == \"/\") print $1}'`"
- name: install additional packages
  yum: name={{ item }} state=present
  with_items:
  - kvm 
  - libvirt 
  - bridge-utils
  - libguestfs-tools
- name: Copy over VSC QCow2
  copy: src=files/software/vsc_singledisk.qcow2 dest=/tmp/vsc_singledisk.qcow2 owner=qemu group=qemu
- name: Copy across XML Template
  copy: src=templates/vsc.xml dest=~/vsc.xml

if i change to 
- name: Copy across XML Template
  copy: src=vsc.xml dest=~/vsc.xml

with absolute paths or not, no difference.
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`

**Comment by willthames** [comment]
Please use triple quotes to surround code - otherwise playbooks get very badly formatted!

**Comment by willthames** [comment]
ansible-lint won't detect missing template or copy sources, if that's what you're expecting.

If that's not what you mean, I'm unclear as to the behaviour that you are expecting.

**Comment by christx2** [comment]
yes, that is what i was expecting as it runs with ansible 2.0.x.x  - hence i was hoping it would catch the changes from 1.9 to 2.0.x and flag it.
would be very helpful.

**Comment by christx2** [comment]
yes, that is what i was expecting as it runs with ansible 2.0.x.x  - hence i was hoping it would catch the changes from 1.9 to 2.0.x and flag it.
would be very helpful.
